app/src/main/java/eu/faircode/xlua/ActivityMain.java

```diff
@@ -82,7 +81,7 @@ protected void onCreate(Bundle savedInstanceState) {
             Snackbar snackbar = Snackbar.make(findViewById(android.R.id.content), getString(R.string.msg_no_service), Snackbar.LENGTH_INDEFINITE);
 
             final Intent intent = getPackageManager().getLaunchIntentForPackage("de.robv.android.xposed.installer");
-            if (intent.resolveActivity(getPackageManager()) != null)
+            if (intent == null || intent.resolveActivity(getPackageManager()) != null)
```

Comment:
There are more references to Xposed, like logging: did you check if that works okay?